implementations/rust/ockam/ockam_command/tests/bats/use_cases.bats

```diff
@@ -73,38 +73,395 @@ teardown() {
   fwd=$(random_str)
 
   # Administrator
-  cp1_token=$($OCKAM project ticket --attribute component=control --relay $fwd)
-  ep1_token=$($OCKAM project ticket --attribute component=edge)
-  x_token=$($OCKAM project ticket --attribute component=x)
+  ADMIN_HOME="$OCKAM_HOME"
+  run_success bash -c "$OCKAM project ticket --attribute component=control --relay $fwd > $OCKAM_HOME/control.ticket"
+  run_success bash -c "$OCKAM project ticket --attribute component=edge > $OCKAM_HOME/edge.ticket"
+  run_success bash -c "$OCKAM project ticket --attribute component=x > $OCKAM_HOME/x.ticket"
 
   # Control plane
   setup_home_dir
   run_success "$OCKAM" project import --project-file $PROJECT_PATH
 
-  $OCKAM identity create control_identity
-  $OCKAM project enroll $cp1_token --identity control_identity
-  $OCKAM node create control_plane1 --identity control_identity
-  $OCKAM policy create --at control_plane1 --resource tcp-outlet --expression '(= subject.component "edge")'
-  $OCKAM tcp-outlet create --at /node/control_plane1 --to 127.0.0.1:5000
-  run_success "$OCKAM" relay create "$fwd" --to /node/control_plane1
+  run_success $OCKAM identity create control_identity
+  run_success $OCKAM project enroll "$ADMIN_HOME/control.ticket" --identity control_identity
+  run_success $OCKAM node create control_plane1 --identity control_identity
+  run_success $OCKAM policy create --at control_plane1 --resource tcp-outlet --expression '(= subject.component "edge")'
+  run_success $OCKAM tcp-outlet create --at /node/control_plane1 --to 127.0.0.1:5000
+  run_success "$OCKAM" relay create "$fwd" --at /project/default --to /node/control_plane1
 
   # Edge plane
   setup_home_dir
   run_success "$OCKAM" project import --project-file $PROJECT_PATH
 
   $OCKAM identity create edge_identity
-  $OCKAM project enroll $ep1_token --identity edge_identity
+  $OCKAM project enroll "$ADMIN_HOME/edge.ticket" --identity edge_identity
   $OCKAM node create edge_plane1 --identity edge_identity
   $OCKAM policy create --at edge_plane1 --resource tcp-inlet --expression '(= subject.component "control")'
   $OCKAM tcp-inlet create --at /node/edge_plane1 --from "127.0.0.1:$port_1" --to "$fwd"
   run_success curl --fail --head --max-time 10 "127.0.0.1:$port_1"
 
   ## The following is denied
   $OCKAM identity create x_identity
-  $OCKAM project enroll $x_token --identity x_identity
+  $OCKAM project enroll "$ADMIN_HOME/x.ticket" --identity x_identity
   $OCKAM node create x --identity x_identity
   $OCKAM policy create --at x --resource tcp-inlet --expression '(= subject.component "control")'
   $OCKAM tcp-inlet create --at /node/x --from "127.0.0.1:$port_2" --to "$fwd"
   run curl --fail --head --max-time 10 "127.0.0.1:$port_2"
   assert_failure 28 # timeout error
 }
+
+start_python_server() {
```

Comment:
This test file is already large. Could you please move all the support code, for python, kafka, influxdb/telegraf, to separate files? (maybe you can even create a use-cases directory)

Reply:
Then, maybe, what we can do in the docs repository is to:

 1. Query the `use-case` directory for the last commit which changed anything in that directory.
 2. If that commit is different from the last commit we stored inside the docs repository, we raise an error in the docs repository CI.
 3. Then we fix the docs if necessary and write down the new `ockam` commit that we think is now the correct one for the docs.

Reply:
Currently experimenting on github status badge https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

Reply:
@etorreborre I added a workflow that queries the latest commit of the use-case directory and the documentation repository and compares them against a [registry on the artifacts repository](https://github.com/build-trust/ockam-artifacts/blob/docker/docs_test_data.json), if the registry doesn't have stored the latest commits, it'll alert us on the development channel